presto-google-sheets/src/main/java/io/prestosql/plugin/google/sheets/SheetsClient.java

```diff
@@ -202,7 +202,7 @@ private Credential getCredentials()
     private List<List<Object>> readAllValuesFromSheetExpression(String sheetExpression)
     {
         try {
-            // by default loading up to max 10k columns
+            // by default loading up to max 10k rows if tab name is not provided
```

Comment:
please mention that 10k rows is taken from the first tab. Alternatively remove the comment. 

Also (as separate commit) please use debug for below logging (line 212). Can you please check if there other places like that?

Reply:
I will remove the comment as part of this PR as this info is already covered in the docs. Thanks!